app/controllers/katello/api/v2/products_controller.rb

```diff
@@ -36,7 +37,15 @@ class Api::V2::ProductsController < Api::V2::ApiController
     param_group :search, Api::V2::ApiController
     def index
       options = {:includes => [:sync_plan, :provider]}
-      respond(:collection => scoped_search(index_relation.uniq, :name, :asc, options))
+      products = scoped_search(index_relation.uniq, :name, :asc, options)
+      respond_to do |format|
+        format.csv do
```

Comment:
Could we use rabl for this so we don't have to do this format swapping? Thats always been the point of rabl -- define the resource in one place and then output that resource to whatever format the user requests.